lib/miq_expression.rb

```diff
@@ -1009,11 +1016,11 @@ def self.get_col_type(field)
   DATE_TIME_OPERATORS = config[:date_time_operators]
 
   def self.get_col_operators(field)
-    if field == :count || field == :regkey
-      col_type = field
-    else
-      col_type = get_col_type(field.to_s) || :string
-    end
+    col_type = if field == :count || field == :regkey
+                 field
+               else
+                 get_col_type(field.to_s) || :string
+               end
```

Comment:
I'm OK with that 👍 (more specifically, Matz' auto indent rules are OK with that ;))